modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -79,6 +79,7 @@ platform:
     datacenter: datacenter
     defaultDatastore: datastore
     folder: folder
+    resourcePool: resource_pool
     diskType: thin <6>
     network: VM_Network
     cluster: vsphere_cluster_name <7>
```

Comment:
I saw that here <7> explains the option of resourcePool, not cluster, need to move <7> to the end of option resourcePool.

Reply:
@jinyunma Thanks for catching that. I think I've clarified the difference between resource pool and vSphere cluster now.